### DISCUSSION TITLE
How to use resourceTimelineView

### DISCUSSION THREAD

**Discussion by ClintonEmok** [saade/filament-fullcalendar#72] [Q&A]
I tried following the fullcalendar documentation for create a resource timeline, but it renders nothing at all.
Could you maybe explain how I can get a config snippet which works?

**Comment by saade** [comment]
https://github.com/saade/filament-fullcalendar/pull/66. works for you?

**Reply by ClintonEmok** [reply]
Unfortunately, it doesn't render an calendar anymore. 
But when i inspect the browser i get this type error
![Screenshot_20230313_232654](https://user-images.githubusercontent.com/54935030/224846637-02b1236c-255b-40ea-8191-0dddc503fc93.png)
rror

**Comment by saade** [comment]
I never used this plugin, maybe someone in #fullcalendar channel on discord can help you :)

https://discord.gg/t8v8MD8M
